### PR TITLE
:lipstick: fix light mode and refacto of common page component

### LINKED ISSUE #314: dark and light mode refacto
some section (particulary bg) are not homogen between the light and dark mode. We should refacto and decide what kind of background/color we should use in light and dark mode

### PR DESCRIPTION
**Pull request by MailyLehoux**
resolve #314

### CODE REVIEW

File: src/components/profile/statistics/Statistics.tsx [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
   return (
-    <div
-      className={`${
-        darkMode
-          ? "bg-dark-dark pb-4 pl-4 pr-4 mt-2 flex-col rounded-lg"
-          : "bg-light pb-4 pl-4 pr-4 mt-2 flex-col rounded-lg"
-      }`}
-    >
-      <h2 className="pb-2 pr-2 pt-6 text-2xl">
+    <div className="bg-light-dark dark:bg-dark-dark pb-4 pl-4 pr-4 my-2 p-2 flex-col rounded-lg">
```
Comment (PierreBrethes) [inline src/components/profile/statistics/Statistics.tsx:13]: Seems like in the project, a lot of div like this one are used, could be great to use some styled components. The goal is that we would use exactly the same design for wrappers or buttons or any UI component. Very handful.
https://www.npmjs.com/package/tailwind-styled-components

**Review by PierreBrethes — APPROVED** [review]

File: src/components/profile/statistics/Statistics.tsx (hunk 1 above)
Comment (MailyLehoux) [reply src/components/profile/statistics/Statistics.tsx:13]: Should be very nice to implement it in Skillz

Comment (benjaminbrgs) [reply src/components/profile/statistics/Statistics.tsx:13]: I agree with you @PierreBrethes

**Review by benjaminbrgs — APPROVED** [review]
Great, that makes the global UI really nice 💪🏼

**Pull request merged**